### DISCUSSION TITLE
i want to run a lite half life server on a raspberry

### DISCUSSION THREAD

**Discussion by cokeman8D** [GameServerManagers/LinuxGSM#3235] [Q&A] [answered]
First of all I want to know if this is possible, I want to think it is... so that I can start on trying this
as seen on linuxgsm I guess I'll need to install ubuntu server on the raspberry as a first step, 
then somehow install steam on it and finally installing hldm, I guess I'll be having to install steamCMD to do so, which I didn't get info about yet, neither on how to install nor how to use it
then as a last step, happily enjoy what I did, setting up the server and showing my maps to the world
Hoping someone answer anything, thx!

**Comment by dgibbs64** [comment, marked as answer]
https://docs.linuxgsm.com/support/faq#will-linuxgsm-run-on-raspberry-pi
